### ISSUE TITLE
Can't install obs-ndi in Debian Buster 10

### ISSUE THREAD

**Issue by Goudarz** [Palakis/obs-ndi#468]
When I try and install the obs-ndi in Debian 10 I get the following error:

```
 $ sudo dpkg -i obs-ndi_4.9.0-1_amd64.deb 
(Reading database ... 429870 files and directories currently installed.)
Preparing to unpack obs-ndi_4.9.0-1_amd64.deb ...
Unpacking obs-ndi (4.9.0-1) over (4.9.0-1) ...
dpkg: dependency problems prevent configuration of obs-ndi:
 obs-ndi depends on obs-studio (>= 25.0.7); however:
  Package obs-studio is not installed.

dpkg: error processing package obs-ndi (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 obs-ndi
```

**Comment by walterav1984** [comment]
Normally a ```sudo apt -f install``` would fix it however there is only "obs-studio 22.03" in buster, so you probably have to [build](https://obsproject.com/wiki/install-instructions#debian-based-build-directions) obs-studio (>= 25.07) yourself.

Could you edit your issue title so it contains Debian Buster 10, and correct start/close your error code with three ```?

**Comment by walterav1984** [comment]
Even when building obs-studio 25.07 commit 078ea52 from source on debian 10 which will satisfy the  `obs-ndi depends on obs-studio (>= 25.0.7); however:` and make the obs-ndi 4.9 deb packages installable, it still doesn't work!

The obs-studio 25.07 log won't show any references to obs-ndi and no output or filters are available in the program.

However removing the obs-ndi 4.9 packages and installing the 4.8 packages make obs-ndi load in obs-studio 25.07 and show output and filter options, this is offcourse unsupported.

The latest version that works ootb for debian buster 10 with apt version 22.03 of obs-studio is obs-ndi 4.7 release.
